modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/filterContainer/customFilter/condition.go

```diff
@@ -90,7 +92,29 @@ func (p *CustomFilter) MemberCondition(key string) (*model.SelectCondition, erro
 		selectOptions = append(selectOptions, *model.NewSelectOption(cputil.I18n(p.sdk.Ctx, "choose-yourself"), p.sdk.Identity.UserID).WithFix(true))
 		return selectOptions
 	}())
-	return condition, nil
+	executorCondition.ConditionBase.Placeholder = cputil.I18n(p.sdk.Ctx, "please-choose-executor")
+
+	creatorCondition := model.NewSelectCondition("creator", cputil.I18n(p.sdk.Ctx, "creator"), func() []model.SelectOption {
+		selectOptions := make([]model.SelectOption, 0, len(members)+1)
+		for _, v := range members {
+			selectOptions = append(selectOptions, *model.NewSelectOption(func() string {
+				if v.Nick != "" {
+					return v.Nick
+				}
+				if v.Name != "" {
+					return v.Name
+				}
+				return v.Mobile
+			}(),
+				v.UserID,
+			))
```

Comment:
you can search code in user util, I think there already have a func like: getUserName

Reply:
done